src/NHSD.BuyingCatalogue.Ordering.Services/OrderService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```

Comment:
I don't think we need the following `using` statements:

```c#
using System.Collections.Generic;
using System.Text;
```

Reply:
Removed `using `statements 
```

using System.Collections.Generic;
using System.Text;
```